.github/CONTRIBUTING.md

```diff
@@ -34,17 +34,28 @@ actual development.
  * Pull requests should be atomic; Make one commit for each distinct change, so if a part
  of a pull request needs to be removed/changed, you may simply modify that single commit.
  Due to limitations of the engine, this may not always be possible; but do try your best.
- * Document and explain your pull requests thoroughly. Detail what each commit changes,
- and why it changes it. We do not want to have to read all of you commit names to figure
- out what your pull request is about.
+
+ * Document and explain your pull requests thoroughly. Failure to do so will delay a PR as
+  we question why changes were made. This is especially important if you're porting a PR 
+  from another codebase (i.e. TG) and divert from the original. Explaining with single
```

Comment:
a single comment

Reply:
Meant to be comments since there could be more than one change.

I prefer it when people explain the change one by one

Reply:
Ah

Reply:
Ah makes sense also with "comments".